CONTRIBUTING.rst

```diff
@@ -23,7 +23,7 @@ In order to add a feature to ``python-ndb``:
 - The feature must be documented in both the API and narrative
   documentation (in ``docs/``).
 
-- The feature must work fully on the following CPython versions:  2.7, 3.6
+- The feature must work fully on the following CPython versions:  3.6
```

Comment:
I'd like to drop 3.6 as well since it's unsupported in the python community.
https://devguide.python.org/versions/#unsupported-versions

Reply:
I would also like to do this. However, if it's okay I'd like to do it on a separate PR.

Reply:
See #829